test/airport.spec.js

```diff
@@ -0,0 +1,205 @@
+const { assertEquals } = require("./test-framework");
+const Airport = require("../src/Airport");
+
+function testLandingPlane() {
```

Comment:
But also good idea that Tests are organized into separate functions for each case.